deeppavlov/models/kbqa/entity_detection_parser.py

```diff
@@ -50,45 +79,47 @@ def __call__(self, question_tokens: List[List[str]],
         return entities_batch, types_batch, positions_batch
 
     def tags_from_probas(self, probas):
-        tag_list = ["O-TAG", "E-TAG", "T-TAG"]
         tags = []
         tag_probas = []
         for proba in probas:
             tag_num = np.argmax(proba)
-            if tag_num in [1, 2]:
+            if tag_num in self.et_prob_ind:
                 if proba[tag_num] < self.thres_proba:
                     tag_num = 0
-            tags.append(tag_list[tag_num])
+            tags.append(self.tag_ind_dict[tag_num])
             tag_probas.append(proba[tag_num])
                     
         return tags, tag_probas
 
     def entities_from_tags(self, tokens, tags, tag_probas):
```

Comment:
Please add the function description and arguments as well.

Reply:
Added docstrings